frontend/src/AlertsManagement/AlertsContent.js

```diff
@@ -0,0 +1,212 @@
+// const resolvedAlerts = [2]
+
+// const allAlerts = [0, 1, 2, 3, 4, 5, 6];
+// const unresolvedAlerts = [0, 1, 2, 3];
+// const resolvedAlerts = [4, 5, 6];
+
+// TODO: create constants file. ref: https://stackoverflow.com/questions/39036457/react-create-constants-file.
+const tabLabels = {
+  UNRESOLVED: 0,
+  RESOLVED: 1,
+  ALL: 2,
+};
+
+class AlertsContent extends Component {
+  constructor(props) {
+    super(props);
+    // getAlerts();
+    this.state = {
+      tab: tabLabels.UNRESOLVED,
+      unchecked: unresolvedAlerts.slice(), 
+      checked: resolvedAlerts.slice(), 
+    };
+  }
+
+  a11yProps = (index) => {
+    return {
+      id: `tab-${index}`,
+      'aria-controls': `tabpanel-${index}`,
+    };
+  }
+  
+  handleTabs = (event, newTab) => {
+    this.setState({tab: newTab});
+  };
+
+  handleCheckbox = (value) => {
+    const unchecked = this.state.unchecked.slice();
+    const checked = this.state.checked.slice();
+    const newUnchecked = [...unchecked];
+    const newChecked = [...checked];
+
+    const currentUncheckedIndex = unchecked.indexOf(value);
+    const currentCheckedIndex = checked.indexOf(value);
+
+    if (currentCheckedIndex === -1 && currentUncheckedIndex !== -1) {
+      // Was unresolved and now want to resolve it, second check is a sanity check.
+      newChecked.push(value);
+      newUnchecked.splice(currentUncheckedIndex, 1);
+    } else if (currentUncheckedIndex === -1 && currentCheckedIndex !== -1) {
+      // Was resolved and now want to unresolve it, second check is a sanity check.
+      newUnchecked.push(value);
+      newChecked.splice(currentCheckedIndex, 1);
+    } else {
+      // This should never happen (programmer error).
+      throw new Error("Misplaced alert: " + allAlerts[value]);
+    }
+
+    this.setState({
+      unchecked: newUnchecked,
+      checked: newChecked,
+    });
+
+    // TODO: send POST request to servlet with status change of alert at index = value.
+  };
+
+  render() {
+    const { tab, unchecked, checked } = this.state;
+    const { classes } = this.props;
+    // getAlerts();
+    return (
+      <div className={classes.root}>
+        <Paper>
+          <Tabs 
+            value={tab} 
+            onChange={this.handleTabs}
+            variant="fullWidth"
+            indicatorColor="secondary"
+            textColor="secondary" 
+            aria-label="alert tabs"
+          >
+            <Tab icon={<ErrorIcon />} label="Unresolved" {...this.a11yProps(tabLabels.UNRESOLVED)} />
+            <Tab icon={<DoneIcon />} label="Resolved" {...this.a11yProps(tabLabels.RESOLVED)} />
+            <Tab icon={<AllInboxIcon />} label="All alerts" {...this.a11yProps(tabLabels.ALL)} />
+          </Tabs>
+        </Paper>
+        <TabPanel value={tab} index={tabLabels.UNRESOLVED}>
+          <AlertsList 
+            allAlerts = {allAlerts}
+            alerts={unchecked} 
+            checked={checked}
+            handleToggle={this.handleCheckbox} 
+          />
```

Comment:
this is unchanged but note that the alerts and checked props are coming from the state of the component and not the outside array